unity-client/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogBridge.cs

```diff
@@ -0,0 +1,128 @@
+using DCL;
+using DCL.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AssetCatalogBridge : MonoBehaviour
+{
+    public static bool VERBOSE = false;
+    public static AssetCatalogBridge i { get; private set; }
+
+    private static SceneAssetPackDictionary sceneAssetPackCatalogValue;
+    public static SceneAssetPackDictionary sceneAssetPackCatalog
```

Comment:
I'm a bit confused about this. Either we use a singleton approach and members are not static, or we just create a static class without the need of an instance.

Reply:
It follows the same structure as `CatalogController` I just copy and paste what you have there

Reply:
You are right, it's about time that `CatalogController`gets a refactor hahaha. Dont worry too much then, we will tackle down both of them when we implement a more robust Storage architecture we have in mind. Thanks!